src/api/app/models/concerns/status_checkable.rb

```diff
@@ -2,7 +2,11 @@ module StatusCheckable
   extend ActiveSupport::Concern
 
   included do
-    serialize :required_checks, Array
+    if RailsVersion.is_7_1?
+      serialize :required_checks, type: Array
+    else
+      serialize :required_checks, Array
+    end
```

Comment:
Are you sure passing the coder as name parameter is not working in 7.0?

Reply:
> Are you sure passing the coder as name parameter is not working in 7.0?

I modified the commit to pass the coder as name parameter, also in 7.0.

Reply:
https://app.circleci.com/pipelines/github/openSUSE/open-build-service/24391/workflows/02b593bc-1418-4207-9b68-8d96e3776501/jobs/229683

Reply:
Tests failed. Passing the coder as name parameter is not working in 7.0. Reverting the change...